### PR TITLE
core: add dash-config.h only if macro is defined

### PR DESCRIPTION
**Pull request by kittywhiskers**

### CODE REVIEW

File: src/stacktraces.cpp [hunk 1]
```diff
@@ -2,6 +2,10 @@
 // Distributed under the MIT software license, see the accompanying
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
+#if defined(HAVE_CONFIG_H)
+#include <dash-config.h>
```
Comment (UdjinM6) [inline src/stacktraces.cpp:6]:
```suggestion
#include <config/dash-config.h>
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
Good catch! 👍 

I changed the branch to `develop` to make it mergeable (we don't merge into `master`). Also, the original `include` was broken actually 🙈 we need to fix it too, not just wrap it with `#if`/`#endif`, pls see a suggestion below.

**Comment by kittywhiskers** [comment]
Glad I could help 👍 

I'll keep the `develop` base branch in mind, I was trying to build Dash using CMake and needed to build it without header generation, so I didn't notice that, my bad!

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by xdustinface — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK

**Comment by PastaPastaPasta** [comment]
@kittywhiskers Thanks very much for this PR!

If you have a dash address I'd be glad to send over a tip for this. 

Looking forward to maybe future PRs :D

**Comment by kittywhiskers** [comment]
Glad to have helped!

**Pull request merged**